Upstream: yes
Source: http://bugs.python.org/issue20901
        also http://bugs.python.org/issue19601
Reason: fix test with newer sqlite

# HG changeset patch
# User Benjamin Peterson <benjamin@python.org>
# Date 1394679112 18000
# Node ID dbc9e3ed5e9f1bd11240eaa971f6c75d6a7013b5
# Parent  4d8a9d12edfa73ea43e956f62f6c7453abb61a7f
weaken callback count inequality (closes #20901)

diff --git a/Lib/sqlite3/test/hooks.py b/Lib/sqlite3/test/hooks.py
--- a/Lib/sqlite3/test/hooks.py
+++ b/Lib/sqlite3/test/hooks.py
@@ -162,7 +162,7 @@ class ProgressTests(unittest.TestCase):
             create table bar (a, b)
             """)
         second_count = len(progress_calls)
-        self.assertTrue(first_count > second_count)
+        self.assertGreaterEqual(first_count, second_count)
 
     def CheckCancelOperation(self):
         """

